app/constants/general.ts

```diff
@@ -36,4 +36,11 @@ export default {
     AUTOCOMPLETE_SPLIT_CHARACTERS: ['.', '-', '_'],
     CHANNEL_USER_ROLE: 'channel_user',
     RESTRICT_DIRECT_MESSAGE_ANY: 'any',
+    LICENSE_SKUS: {
```

Comment:
0/5 but I think is best in a `license.ts` constant file

Reply:
Makes sense 👍